src/vendor/wordpress/styles/_core-blocks.scss

```diff
@@ -138,32 +124,53 @@ $wp-button-gap: size.$spacing-gap-button-group-default;
   }
 }
 
-/**
- * Gutenberg block: Image
- * Styles for images inserted via Gutenberg blocks.
- *
- * 1. Since we are applying `margin-top` to CMS content via our o-rhythm
- *    pattern, the default `margin-bottom` value on Gutenberg image blocks is
- *    unnecessary, so we remove it here.
- */
+/// Gutenberg block: Image
+/// Styles for images inserted via Gutenberg blocks.
```

Comment:
I realize our comment styles are mixed now, I just can't stand writing the old CSS versions by hand when the Sass comments come with a bunch of automatic niceties in VS Code. I've been slowly trying to write more and more in that style for this project.